commands.py

```
@@ -1,16 +1,274 @@
+            WAITING_CHANNEL: [MessageHandler(filters.TEXT & ~filters.COMMAND, receive_channel)],
+            WAITING_MESSAGE: [MessageHandler(filters.TEXT & ~filters.COMMAND, receive_message)],
+            WAITING_HOUR: [MessageHandler(filters.TEXT & ~filters.COMMAND, receive_hour)],
+        },
+        fallbacks=[CommandHandler("cancel", cancel)],
+    )
+
+    app.add_handler(CommandHandler("start", start))
+    app.add_handler(CommandHandler("stop", stop))
+    app.add_handler(conv_handler)
+
+    # بازیابی jobها و شروع scheduler
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)